packages/linenoise/linenoise.0.9.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+homepage: "https://github.com/fxfactorial/ocaml-linenoise"
+bug-reports: "https://github.com/fxfactorial/ocaml-linenoise/issues"
+license: "BSD-3-clause"
+dev-repo: "https://github.com/fxfactorial/ocaml-linenoise/.git"
```

Comment:
There's an extra '`/`' here.  This line should be:

`dev-repo: "https://github.com/fxfactorial/ocaml-linenoise.git"`